Book-A-Desk.Domain/CommandHandler.Reservations.fs

```diff
@@ -1,36 +1,26 @@
 namespace Book_A_Desk.Domain.CommandHandler
 
 open Book_A_Desk.Core
-open Book_A_Desk.Domain
 open Book_A_Desk.Domain.Events
 open Book_A_Desk.Domain.Reservation
 open Book_A_Desk.Domain.Reservation.Domain
 open Book_A_Desk.Domain.Reservation.Commands
 
 type ReservationsCommandHandler =
     {
-        Handle: ReservationCommand -> Result<unit,string>
+        Handle: ReservationCommand -> Result<DomainEvent list,string>
     }
-
-module ReservationsCommandHandler =
-   let provide (eventStore:EventStore) reservationCommandsFactory =
-
+    
+module ReservationsCommandHandler =  
+   let provide events reservationCommandsFactory =       
        let handle (command : ReservationCommand) =
```

Comment:
The fetch and store is now done by the API, no the Domain, since these are asynchronous operations